doc/User_profile.json

```diff
@@ -79,56 +79,35 @@
 	],
 	"picture": "https://s.gravatar.com/avatar/...png",
 	"shirtSize": "L",
-	"membership": {
-		"Workday": [
+	"groups": [
```

Comment:
note @johngian due to this i believe no change to the PR is needed
this schema was discussed a couple of weeks back (with you :) and is also explained above

I can also add this in the readme though. noter that https://github.com/mozilla-iam/cis/issues/48 references all this